microsoftgraph/microsoft-graph-explorer-v4
# Fix: Show correct information on Modify Permissions tab

## Overview
Solves #1456 

### Demo
![image](https://user-images.githubusercontent.com/45680252/156172594-44b4b9ad-215b-4b46-bda1-9fa4af722895.png)

![image](https://user-images.githubusercontent.com/45680252/156172819-90836e51-5c58-472c-9419-4048a6ee66b8.png)

### Notes
## Testing Instructions
Sign in to Graph Explorer
Click on Modify Permissions tab
Notice that permissions required for the current query are loaded
On the settings menu (...) click on Select Permissions
Notice that the modify permissions tab is still displaying correct information even with the permissions panel open
